contrib/remotexact/rwset.c

```diff
@@ -232,8 +237,8 @@ RWSetToString(RWSet *rwset)
 	/* Header */
 	header = &rwset->header;
 	appendStringInfoString(&s, "{\n\"header\": ");
-	appendStringInfo(&s, "{ \"dbid\": %d, \"xid\": %d, \"region_set\": %ld }",
-					 header->dbid, header->xid, header->region_set);
+	appendStringInfo(&s, "{ \"dbid\": %d, \"xid\": %d, \"csn\": %d, \"region_set\": %ld }",
```

Comment:
Use `%ld`. Change the per-page csn below, too.

Reply:
Done.